hasjob/templates/layout.html.jinja2

```diff
@@ -229,5 +240,20 @@
 
 {% block layoutscripts %}
   {%- if header_campaign %}{{ campaign_script() }}{% endif %}
+  <script src='{{ asset_path('manifest') | safe }}' type="text/javascript"></script>
+  <script src='{{ asset_path('vendor') | safe }}' type="text/javascript"></script>
+  <script src='{{ asset_path('app') | safe }}' type="text/javascript"></script>
```

Comment:
Not safe to use the `| safe` filter. If `asset_path` is confirming the strings are escaped, let it return `Markup` strings.

Reply:
Changed